### ISSUE TITLE
Environment variable expansion is not applied to schemaPath

### ISSUE THREAD

**Issue by erydo** [kamilkisiela/graphql-config#46]
This prevents use-cases such as:
```json
"projects": {
  "foo-service": {
    "schemaPath": "${env:FOO_SERVICE_SCHEMA_PATH}"
  }
}
```
Which is useful for sharing between local dev and CI environments.

**Comment by schickling** [comment]
I agree, this should be possible. @erydo would you mind creating a PR for this including some test cases?

**Comment by kamilkisiela** [comment]
Hi there, it's [The Guild](http://the-guild.dev/) :) We're working on a new version of GraphQL Config. The goal is to make it platform agnostic, flexible enough to use with any tool and framework, reduce the bundle size and a lot more and what's even more important, bring it back to life and keep maintaining.

New changes #162 are already merged into master and `v3.0.0-alpha.13` is released.

With new GraphQL Config you're able to use environment variables like this:


```yaml
schema: ${MY_SCHEMA:schema.graphql}
```

Where `MY_SCHEMA` is the value coming from `process.env.MY_SCHEMA` and `schema.graphql` is a default value
